buddyboss/buddyboss-platform
# PROD-764 Document is not opening in popup for network search - Fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #2780  .

### How to test the changes in this Pull Request:

1. Search for a document for header search bar
2. Client on the document
3. See the document is opening in popup

### Proof Screenshots or Video

https://www.loom.com/share/bf88a1611efd4fed934608c6e85ac83a

### Other information:




### Changelog entry

> Document opening issue network search - fixed
